data/korath/korath variants.txt

```diff
@@ -0,0 +1,714 @@
+# Copyright (c) 2014-2016 by Michael Zahniser
```

Comment:
Technically speaking, MZ *did* create this file, it's just been renamed and had bits taken out, and the file history shows this.  While simultaneously, thanks to some git magic, not being renamed and having other bits taken out.  Like some sort of Schrodinger's text file.

Also it's purely stat blocks, not really something I care to attach copyright to.  If anything I'd want to be on the other files, for all the description work I've been doing.

Reply:
Fair enough. That being said you in particular and others to various extent *have* put a lot of work into it, I'd still suggest adding an additional copyright line reflecting this. And to the other files, too. This is the biggest addition to the Korath since, well, 2016ish. It'd be beneficial to have it recorded that a bunch of work on them did in fact take place in 2022.

As far as that goes, I'd say it should be perfectly acceptable to even just have add one line to represent modern efforts:

```suggestion
# Copyright (c) 2014-2016 by Michael Zahniser
# Copyright (c) 2022 by Ravenshining, EjoThims, Saugia, and Zitchas
```

(sorry, I'm sure I'm not sure who has had a role in making variants)

Reply:
If you insist on crediting me, in ES files I go by Lia Gerty, not Ravenshining

Reply:
I have a lot of respect for the amount of work you are doing here. So yes, I think I'll insist on crediting you. Updated that.